src/main/scala/io/iohk/ethereum/db/storage/KnownNodesStorage.scala

```diff
@@ -14,9 +14,9 @@ class KnownNodesStorage(val dataSource: DataSource) extends KeyValueStorage[Stri
   val key = "KnownNodes"
 
   val namespace: IndexedSeq[Byte] = Namespaces.KnownNodesNamespace
-  def keySerializer: String => IndexedSeq[Byte] = _.getBytes
-  def valueSerializer: Set[String] => IndexedSeq[Byte] = _.mkString(" ").getBytes
-  def valueDeserializer: IndexedSeq[Byte] => Set[String] = (valueBytes: IndexedSeq[Byte]) => new String(valueBytes.toArray).split(' ').toSet
+  def keySerializer: String => Array[Byte] = _.getBytes
+  def valueSerializer: Set[String] => Array[Byte] = _.mkString(" ").getBytes
+  def valueDeserializer: Array[Byte] => Set[String] = (valueBytes: Array[Byte]) => new String(valueBytes).split(' ').toSet
```

Comment:
It could be written better without conversion to String.

Reply:
Do you mean something like this: ```valueBytes.toSet.map(_.toString)```?

Reply:
Do you mean something like this: `valueBytes.toSet.map(_.toString)`?

Reply:
No. I have in mind something like that:
```
var result = Set.empty[String]
var (x, y) = valueBytes.span(_ != ' ')
result += x.mkString
y = y.drop(1)

while(y.nonEmpty){
  val (b, c) = y.span(_ != ' ')
  result += b.mkString
  y = c.drop(1)
}
```
But I am not sure if it is better solution ;)